sr_robot_lib/include/sr_robot_lib/MST.hpp

```diff
@@ -21,6 +21,7 @@
 
 #include <vector>
 #include <string>
+#include <memory>
```

Comment:
Do you have the error output, or a way to run the linter locally? Including an additional lib doesn't feel very linter-y

Reply:
It must be "Include what you use" error for `std::shared_ptr`.

Reply:
I can run it locally, but this is the error given by the linter: 
`Error 2 in /root/workspace/src/codebuild/output/src417757905/src/github.com/shadow-robot/sr-ros-interface-ethercat/sr_robot_lib/include/sr_robot_lib/MST.hpp:66:  Add #include <memory> for shared_ptr<>  [build/include_what_you_use]`

Reply:
Ahh ok, thanks all